unit-tests-java8/src/test/java/org/eclipse/collections/test/map/MapIterableTestCase.java

```diff
@@ -34,7 +35,7 @@
 import static org.junit.Assert.assertThrows;
 import static org.junit.Assert.assertTrue;
 
-public interface MapIterableTestCase extends RichIterableWithDuplicatesTestCase
+public interface MapIterableTestCase extends RichIterableWithDuplicatesTestCase, MapTestCase
```

Comment:
This is a tricky detail. Every MapIterable in Eclipse Collections implements Map, even the immutable ones. That means we ought to be running tests against methods like Map.put() for every MapIterable. This is the core of the change that led to discovering so many inconsistencies between Mutable, Immutable, and Unmodifiable maps.